pyomo/contrib/solver/common/base.py

```diff
@@ -396,9 +396,27 @@ class LegacySolverWrapper:
     interface. Necessary for backwards compatibility.
     """
 
+    class _all_true(object):
```

Comment:
Nope, I reserve the right to tell you to remove `(object)` here because it is nowhere else in this file.